WordPressAuthenticator/Extensions/UIImage+Assets.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
```

Comment:
I really like this extension. Cleans up the code nicely.

Reply:
Thank you Steph!!